lib/sqlbuilder/builder_test.go

```diff
@@ -349,11 +349,16 @@ func TestSelect(t *testing.T) {
 	)
 
 	assert.Equal(
-		`SELECT * FROM "artist" LIMIT -1 OFFSET 5`,
```

Comment:
How about generic `db.Reset` that could be used for limit, offset, columns and so on?